sample_input_files/README.md

```diff
@@ -1,51 +1,71 @@
 This is essentially a weighted edge list.
 * First column is the source location label
 * Second column is the destination location label
-* Third column is the number of individuals undertaking that journey as reported in wu01uk
+* Third column is the number of individuals undertaking that journey as
+  reported in wu01uk
+* Fourth column is a multiplier that gives control over how much dampening
+  factors impact this edge
 
-## sample_20200327_comix_social_contacts.csv
-This is a possible square matrix of mixing - each column and row header is an age category. For a simplified version, look at simplified_age_infection_matrix_from_comix.csv.  It is from https://cmmid.github.io/topics/covid19/comix-impact-of-physical-distance-measures-on-transmission-in-the-UK.html, and includes all imputed contacts (both face-to-face conversation and physical contact).
+## human/full-mixing-matrix/1/data.csv
 
-## simplified_age_infection_matrix_from_comix.csv
-This is a sample *simplified* square matrix describing age mixing between aggregated age categories, generated from sample_20200327_comix_social_contacts.csv by summing over the columns that need to be aggregated, and then summing (weighted by the approximate population in each category) over the rows that need to be aggregated.  This includes both face-to-face and physical contacts, and is not yet down-weighted to approximate infection probability.  
-Age proportion estimated from https://www.ons.gov.uk/peoplepopulationandcommunity/populationandmigration/populationestimates/articles/overviewoftheukpopulation/august2019.
+This comes from is from
+https://cmmid.github.io/topics/covid19/comix-impact-of-physical-distance-measures-on-transmission-in-the-UK.html,
+and includes all imputed contacts (both face-to-face conversation and physical
+contact). This is the entire comix matrix, for all age groups, which is not
+compatible with the population data we have this far. Look the
+_human/mixing-matrix/1/data.csv_ below.
 
+## human/mixing-matrix/1/data.csv
 
-## movement_multipliers.csv
+This is a sample *simplified* square matrix describing age mixing between
+aggregated age categories, generated from _human/full-mixing-matrix/1/data.csv_
+by summing over the columns that need to be aggregated, and then summing
+(weighted by the approximate population in each category) over the rows that
+need to be aggregated.  This includes both face-to-face and physical contacts,
+and is not yet down-weighted to approximate infection probability.  Age
+proportion estimated from
+https://www.ons.gov.uk/peoplepopulationandcommunity/populationandmigration/populationestimates/articles/overviewoftheukpopulation/august2019.
 
-This table has two columns: Time and Movement Multiplier. For instance, if the file looks like this:
+## human/movement-multpliers/1/data.csv
+
+This table has two columns: Time and Movement Multiplier. For instance, if the
```

Comment:
The table has three columns I think?
```{csv}
Time,Movement_Multiplier,Contact_Multiplier
1,0.187005174,0.8
2,0.170162037,0.8
3,0.159949074,0.8
...,...,...
```

Reply:
https://github.com/ScottishCovidResponse/simple_network_sim/blob/05e4b48abe9f431d436aaf0491c92f1ae288d96e/sample_input_files/human/movement-multipliers/1/data.csv